delfin/drivers/fake_storage/__init__.py

```diff
@@ -131,6 +131,55 @@ def get_storage(self, context):
             'subscribed_capacity': subscribed
         }
 
+    @staticmethod
+    def get_spec():
+        sn = six.text_type(uuidutils.generate_uuid())
+        return {
+            'name': 'fake_driver',
+            'description': 'fake driver.',
+            'vendor': 'fake_vendor',
+            'model': 'fake_model',
+            'status': 'normal',
+            'serial_number': sn,
+            'is_historic': False,
+            'resourceMetrics': {
```

Comment:
The resource metrics can be taken from a common template.. So that we enforce the driver to follow common names.. 
We can probaly have a json to represnt the template.. drivers can map that json to dictionary and output only keys which are supported by driver.. just a thought!!!

Reply:
The current implementation is as per your suggestion. Template is defined in "driver_spec_schema.py" file